src/client/utils/tracking/gtm.ts

```diff
@@ -23,7 +23,7 @@ type GTMUserProperties = {
   environment: AppEnvironment
 }
 
-type GTMOfferData = {
+export type GTMOfferData = {
```

Comment:
I don't this being imported anywhere else so we should probably not export it here :)

Reply:
Good catch!